package/MDAnalysis/core/groups.py

```diff
@@ -3324,6 +3324,66 @@ def write(self, filename=None, file_format=None,
 
         raise ValueError("No writer found for format: {}".format(filename))
 
+    def sort(self, key='ix', keyfunc=None):
+        """Return stably sorted Atomgroup by a key that specifies
+           an attribute of atomgroup. The default key is "ix".
+
+        Parameters
+        ----------
+        keys: str
+            The name for attribute of Atomgroup. (e.g. "ids", "ix")
+        keyfunc: function
+            A function that returns 1 dimension array, a key for sorting the
+            atomgroup from multiple dimension array that the attribute returns.
+
+        Returns
+        -------
+        output:
+            Sorted atomgroup.
+
+        Example
+        -------
+
+        .. code-block:: python
+
+            >>> import MDAnalysis as mda
+            >>> from MDAnalysisTests.datafiles import PDB_small
+            >>> u = mda.Universe(PDB_small)
+            >>> ag = u.atoms[[3, 2, 1, 0]]
+            >>> ag.ids
```

Comment:
In my opinion, the inclusion of both `ids` and `ix` in this docstring example is confusing for users. It feels like it hints that the sort method sorts by `ids` because that's the attribute you are showing off first. For the sake of simplicity, I would suggest just having `ag.ix`.

Reply:
Fixed.
https://github.com/MDAnalysis/mdanalysis/pull/3188/commits/a577298b4985b533233bc52a3a7bf75323055312